util/monitoring.js

```diff
@@ -75,6 +75,30 @@ async function getNetworkReceivedInfo(client, projectId, instanceName, startTime
   return requestAndAssembleInfo(client, request);
 }
 
+async function getMemoryUsageInfo(client, projectId, instanceName, startTime, endTime) {
```

Comment:
What about using 'percent' in the function name (e.g. 'getMemoryPercentInfo()') following google API's original terms ('percent_used')?

Reply:
And 'utilization' for cpu (e.g. getCpuUtilizationInfo())?

Reply:
Looks good! Done